tests/ArmTemplates.Tests/Extractor/Scenarios/TagExtractorTests.cs

```diff
@@ -129,5 +129,74 @@ public async Task GenerateTagTemplates_GetApiOperationRelated_ProperlyLaysTheInf
             resources.Tags.Any(x => x.Name.Contains(MockTagClient.OperationTagName1)).Should().BeTrue();
             resources.Tags.Any(x => x.Name.Contains(MockTagClient.OperationTagName2)).Should().BeTrue();
         }
+
+        [Fact]
+        public async Task GenerateTagTemplates_GetApiRelatedTags_ProperlyLaysTheInformation()
+        {
+            // arrange
+            var currentTestDirectory = Path.Combine(this.OutputDirectory, nameof(GenerateTagTemplates_GetApiRelatedTags_ProperlyLaysTheInformation));
+
+            var extractorConfig = this.GetDefaultExtractorConsoleAppConfiguration();
+            var extractorParameters = new ExtractorParameters(extractorConfig);
+
+            var mockedTagClient = MockTagClient.GetMockedApiClientWithDefaultValues();
+            var tagExtractor = new TagExtractor(
+                this.GetTestLogger<TagExtractor>(),
+                mockedTagClient,
+                new TemplateBuilder());
+
+            var extractorExecutor = ExtractorExecutor.BuildExtractorExecutor(
+                this.GetTestLogger<ExtractorExecutor>(),
+                tagExtractor: tagExtractor);
+            extractorExecutor.SetExtractorParameters(extractorParameters);
+
+            ApiTemplateResources apiTemplateResources = new ApiTemplateResources()
+            {
+                Tags = new List<TagTemplateResource>()
```

Comment:
nit: remove curly braces if you still use {} for object initialization?

Reply:
and lower also

Reply:
thank you! Agree updated!